### PR TITLE
Handle file update message

### PR DESCRIPTION
**Pull request by andrecardoso**
Adds basic handling of MSFILEUPDATE message. Still needs to be manually tested.

### CODE REVIEW

File: src/messaging/messaging-service-client.js [hunk 1]
```diff
@@ -16,17 +18,44 @@ function connect(displayId, machineId) {
   connection.on('close', () => console.log('MS connection closed'));
   connection.on('end', () => console.log('MS disconnected'));
   connection.on('error', (error) => logger.error('MS connection error', error));
-  connection.on('data', (data) => console.log(`MS received data: ${JSON.stringify(data)}`));
+  connection.on('data', (data) => {
+    console.log(`MS received data: ${JSON.stringify(data)}`);
+
+    const message = data.topic || data;
+    const handlers = messageHandlers[message.toLowerCase()];
```
Comment (tejohnso) [inline src/messaging/messaging-service-client.js]: Could add a guard against message not being a string here.

Comment (andrecardoso) [reply src/messaging/messaging-service-client.js]: Good call.

File: src/storage/storage.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const messagingServiceClient = require('../messaging/messaging-service-client');
+const fileDownloader = require('./file-downloader');
+
+function processUpdate(message) {
+  const {filePath, version, token} = message;
+  console.log(`Received updated version ${version} for ${filePath}`);
+  console.log(`Token timestamp ${token.data.timestamp}`);
+
+  return fileDownloader.download({filePath, version, token})
```
Comment (tejohnso) [inline src/storage/storage.js]: Shouldn't there also be a file metadata update here?

Comment (andrecardoso) [reply src/storage/storage.js]: Yes, but I tried to simplify for this first PR. I'm hesitant to add the lokijs db library. What do you think?

Comment (tejohnso) [reply src/storage/storage.js]: Calls that don't do anything, or comments would be ok.
I'm ok with lokijs. Just check for any github issues mentioning chrome os :)

Comment (santiagonoguez) [reply src/storage/storage.js]: I wouldn't add lokijs, if there's native indexeddb support. Even with that, maybe local storage or something simpler could be better.

Comment (santiagonoguez) [reply src/storage/storage.js]: or if it has no issues with the browser, as Tyler mentions, could be. It's simpler than indexeddb

Comment (andrecardoso) [reply src/storage/storage.js]: Done. I've created an empty DB object that gets called on msfileupdate.

Comment (andrecardoso) [reply src/storage/storage.js]: @tejohnso @santiagonoguez  Please review it again. Thanks

File: src/storage/db.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const fileMetadata = {
+  put(entry) {
+    console.log('to be implemented', entry);
```
Comment (santiagonoguez) [inline src/storage/db.js]: shouldn't these return promises ?

Comment (andrecardoso) [reply src/storage/db.js]: Yes, good call.

Comment (tejohnso) [reply src/storage/db.js]: I'm pretty sure loki operates in-memory and isn't returning promises. If that's the case, I don't think it makes sense to return a promise. Can be confirmed later.

**Pull request merged**